cts/scheduler/summary/not-reschedule-unneeded-monitor.summary

```diff
@@ -11,14 +11,15 @@ Current cluster status:
     * Clone Set: c-vm-fs [vm1]:
       * Started: [ castor kimball ]
     * xen-f	(ocf:heartbeat:VirtualDomain):	 Stopped (disabled)
-    * sle12-kvm	(ocf:heartbeat:VirtualDomain):	 FAILED castor
+    * sle12-kvm	(ocf:heartbeat:VirtualDomain):	 FAILED [ kimball castor ]
     * Clone Set: cl_sgdisk [sgdisk]:
       * Started: [ castor kimball ]
 
 Transition Summary:
-  * Recover    sle12-kvm     ( castor -> kimball )
+  * Recover    sle12-kvm     ( kimball )
 
 Executing Cluster Transition:
+  * Resource action: sle12-kvm       stop on kimball
```

Comment:
I don't understand, why would we need to stop on kimball when there's already a successful stop as the most recent operation?